lib/repository.py

```diff
@@ -23,7 +23,7 @@
 
 from lib import config
 from lib import exception
-
+from lib.utils import run_command
```

Comment:
This is also being imported above in `import utils` and is being used in two different formats. Could you make it consistent and move the import to the correct section?